scripts/macpython-build-module-wheels.sh

```diff
@@ -62,7 +62,7 @@ VENVS=()
 source "${script_dir}/macpython-build-common.sh"
 # -----------------------------------------------------------------------
 
-if test -e setup.py; then
+if test -e pyproject.toml; then
```

Comment:
This is not correct.

Reply:
This is a check to see if the module has a setup.py file for modules that have not been migrated to pyproject.toml yet, and it builds for setup.py or pyproject.toml accordingly. This will break modules that have not migrated from setup.py to pyproject.toml.